docs/forms.md

```diff
@@ -0,0 +1,145 @@
+# OneBlink SDK | Forms
+
+## Constructor
+
+| Parameter | Required | Type | Description
+|---|---|---|---|
+| `options.accessKey` | Yes | `string` | Access key provided by OneBlink. |
+| `options.secretKey` | Yes | `string` | Secret key provided by OneBlink. |
+| `options.formsRendererOrigin` | No | `string` | The Origin of the Forms Renderer. Should only be used for development and testing purposes. |
```

Comment:
Should we even expose these options to them or leave them undocumented?

Reply:
im easy, what do you think?

Reply:
Undocumented